src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -230,7 +230,7 @@ private bool FeaturesEnabled()
                     return true;
                 }
 
-                return false;
+                return true;
```

Comment:
I've reverted this.  This will now not run out of proc until we decide to turn OOP on.